### PR TITLE
Changelog for 3.x

### PR DESCRIPTION
**Pull request by NicolasCARPi**
Should be the last PR before tagging 3.0.0!

### CODE REVIEW

**Review by RobThree — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,35 @@
...
+
+Documentation on selecting a QR Code Provider is available here: [QR Code Provider documentation](https://robthree.github.io/TwoFactorAuth/qr-codes.html).
+
+### Default secret length
+
+The default secret length has been increased from 80 bits to 160 bits (RFC4226) PR #117. This might cause an issue in your application if you were previously storing secrets in a column with restricted size. This change doesn't impact existing secrets, only new ones will get longer.
+
+Previously a secret was 16 characters, now it needs to be stored in a 32 characters width column.
+
+You can keep the old behavior by setting `80` as argument to `createSecret()` (not recommended).
```
Comment (willpower232) [inline CHANGELOG.md:33]:
```suggestion
You can keep the old behavior by setting `80` as argument to `createSecret()` (not recommended, see [#117](https://github.com/RobThree/TwoFactorAuth/pull/117) for further discussion).
```

**Review by willpower232 — APPROVED** [review]
Some suggestions but seems fine to me.

@NicolasCARPi FYI I presume that the assignee should remain as yourself since the individual responsible for the work?

**Comment by NicolasCARPi** [comment]
>  the assignee

You'r talking about this?

2024-04-28-002408_333x124_scrot

I'm not sure what is the difference between Assignees and Reviewers here...

**Comment by willpower232** [comment]
Yeah, you can request reviews to get people on the reviewers list and then assign yourself as the assignee being the person responsible for working on the PR. There doesn't seem to be any hard and fast rules about it so mostly I see it getting ignored 😅

**Pull request merged**